README.md

```diff
@@ -4,11 +4,11 @@
 
 ## Overview
 
-This is a Kubernetes cluster that includes logging, monitoring, and operational
+This is a Kubernetes 1.4.0 cluster that includes logging, monitoring, and operational
```

Comment:
We have a badge that we're updating with the kubernetes version number. I dont know that we wish to include this inline as its another source to update.